tutor/src/flux/step-title.coffee

```diff
@@ -7,9 +7,14 @@ TEXT_LENGTH_CHECK = 110
 TEXT_LENGTH = 125
 TEXT_CHECK_RANGE = TEXT_LENGTH - TEXT_LENGTH_CHECK
 
+isLearningObjectives = (element) ->
+  (element?.attribs?['class']? and
+    element.attribs['class'].search(/learning-objectives/) > -1) or
+    element.attribs['data-type'] is 'abstract'
+
 isTypedClass = (element) ->
   element?.attribs?['class']? and
-    element.attribs['class'].search(/learning-objectives|references|ap-connection/) > -1
+    element.attribs['class'].search(/learning-objectives|references|ap-connection|solution/) > -1
 
```

Comment:
can we use classList yet?  http://caniuse.com/#search=classlist seems to say it's ok, but I think maybe we had bugs with it once?